.github/workflows/lintdocs.yml

```diff
@@ -0,0 +1,108 @@
+name: "Lint P4CP Documentation"
+
+on:
+  push:
+    branches:
+      - main
+      - ipdk_v*
+  pull_request:
+    branches:
+      - main
+      - ipdk_v*
+
+permissions: read-all
+
+jobs:
+  #---------------------------------------------------------------------
+  # 1-markdownlint
+  #---------------------------------------------------------------------
+  markdownlint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+      - name: Clone networking-recipe
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get changed markdown files
+        id: changed
+        uses: tj-actions/changed-files@v41
+        with:
+          files: |
+            **.md
+
+#     - name: List all changed markdown files
```

Comment:
Remove commented section?

Reply:
It's a troubleshooting aid. It took me a lot of tries to get the conditional file selection logic to work, and I want to hang on to the debug code a bit longer (and capture a copy of it in the commit history) before removing it.